dev-resources/less/grid.less

```diff
@@ -186,17 +186,17 @@ border: 1px solid #afafaf background-header: #c8c8c8 background-odd: #fafafa bac
     position: relative;
   }
   tr.parillinen {
-    background-color: @harmaa9;
+    background-color: @harmaa10;
```

Comment:
Eräässä commitissa tuli lisättyä lisää harmaan sävyjä. Sen takia piti muuttaa näitä olemassaolevien harmaiden nimiä.
Harmaa9->harmaa10
harmaa10->harmaa15
harmaa11->harmaa13

Meillä oli itseasiassa aiemmin moka, että harmaa11 oli tummempi kuin harmaa10. Nyt tuli sekin korjattua.